### PR TITLE
Xraytest

### PR DESCRIPTION
**Pull request by JohnWu20**
Add config path

### CODE REVIEW

File: terraform/testcases/xrayreceiver/parameters.tfvars [hunk 1]
```diff
@@ -1,8 +1,15 @@
+otconfig_path="../template/otconfig/xrayreceiver_otconfig.tpl"
```
Comment (wyTrivail) [inline terraform/testcases/xrayreceiver/parameters.tfvars]: you can remove this.

File: terraform/testcases/xrayreceiver/parameters.tfvars [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+ecs_taskdef_path="../template/ecstaskdef/xrayreceiver_ecs_taskdef.tpl"
```
Comment (wyTrivail) [inline terraform/testcases/xrayreceiver/parameters.tfvars]: you can remove this

File: terraform/testcases/xrayreceiver/parameters.tfvars [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
 
 # sample application image to emit the trace data
-data_emitter_image="johnwu20/sample_server_app:v1.0.0"
+data_emitter_image="johnwu20/sample_server_app:v1.0.1"
 
 # AOC image
 aoc_image_repo="johnwu20/aocimage"
+
+# todo this version needs to be removed, instead version should be received from workflow
+aoc_version="v0.1.1"
```
Comment (wyTrivail) [inline terraform/testcases/xrayreceiver/parameters.tfvars]: you can remove this

File: terraform/testcases/xrayreceiver/parameters.tfvars [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+
 # this file is defined in validator/src/main/resources/validations
 validation_config="default-xray-trace-validation.yml"
 
 # sample application image to emit the trace data
-data_emitter_image="johnwu20/sample_server_app:v1.0.0"
+data_emitter_image="johnwu20/sample_server_app:v1.0.1"
 
 # AOC image
 aoc_image_repo="johnwu20/aocimage"
```
Comment (wyTrivail) [inline terraform/testcases/xrayreceiver/parameters.tfvars]: you can remove this

**Review by wyTrivail — CHANGES_REQUESTED** [review]

File: terraform/testcases/xrayreceiver/parameters.tfvars (hunk 4 above)
Comment (JohnWu20) [reply terraform/testcases/xrayreceiver/parameters.tfvars]: OK, will delete it

File: terraform/testcases/xrayreceiver/parameters.tfvars (hunk 3 above)
Comment (JohnWu20) [reply terraform/testcases/xrayreceiver/parameters.tfvars]: Removed

**Review by wyTrivail — APPROVED** [review]

**Pull request closed without merging**